integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -273,16 +347,17 @@ def export(self, **kwargs):
                 for item in content_metadata_items:
                     key = get_content_metadata_item_id(item)
 
-                    # Used to save modified times and catalog uuid's to sent transmission records
-                    content_updated_mapping[key] = {
-                        'modified': item.get('content_last_modified'),
-                        'catalog_uuid': enterprise_customer_catalog.uuid
-                    }
-
                     # transform the content metadata into the channel specific format
                     transformed_item = self._transform_item(item)
                     if key in items_create_keys:
                         create_payload[key] = transformed_item
+                        existing_record = items_to_create.get(key)
+                        existing_record.channel_metadata = transformed_item
+                        existing_record.content_last_changed = item.get('content_last_modified')
+                        # Sanity check
+                        existing_record.enterprise_customer_catalog_uuid = enterprise_customer_catalog.uuid
+
+                        create_payload[key] = existing_record
```

Comment:
if we have this, is [line 353](https://github.com/openedx/edx-enterprise/pull/1593/files#diff-bd1856bd7ffab338eb216ef940de368327f49152f8673cc007eac15557220c78R353) still necessary?

Reply:
great catch